physics/CONV/Chikira_Sugiyama/cs_conv.F90

```diff
@@ -3668,26 +3837,28 @@ END SUBROUTINE CUMDNR
 !***********************************************************************
 !>\ingroup cs_scheme
       SUBROUTINE CUMSBR                                      & !! Tracer Subsidence
-                      ( IM    , IJSDIM, KMAX  , NTR   ,      & !DD dimensions
+                      ( IM    , IJSDIM, KMAX, NTR, NCTP,     & !DD dimensions
                         GTR   ,                              & ! modified
-                        GDR   , DELPI ,                      & ! input
+                        GDR   , DELP  ,                      & ! input
                         GMFLX , KTMX  , OTSPT ,              & ! input
+                        sigmai        , sigma ,              & !DDsigma input
                         ISTS, IENS )                           ! input
 !
       IMPLICIT NONE
 
-      INTEGER, INTENT(IN) :: IM, IJSDIM, KMAX, NTR             !! DD, for GFS, pass in
+      INTEGER, INTENT(IN) :: IM, IJSDIM, KMAX, NTR, nctp       !! DD, for GFS, pass in
 !
 !   [MODIFY]
       REAL(kind_phys)     GTR   (IJSDIM, KMAX, NTR)   !! tracer tendency
```

Comment:
@AnningCheng-NOAA The code in https://github.com/ufs-community/ccpp-physics/pull/171 wouldn't compile without adding the `NTR` dimension back to `GTR` since the third dimension of `GTR` is used in this subroutine. Please double-check that this is OK. This is the only **CODE** change in https://github.com/ufs-community/ccpp-physics/pull/171 that was reverted.

Reply:
yes, PLease adding NTR back.